Extensions/TweenBehavior/JsExtension.js

```diff
@@ -198,6 +198,64 @@ module.exports = {
       .getCodeExtraInformation()
       .setFunctionName("addObjectPositionXTween");
 
+    behavior
+      .addAction(
+        "addObjectWidthTween",
+        _("Add object width tween"),
+        _("Add a tween animation for the object width."),
+        _(
+          "Tween the width of _PARAM0_ to _PARAM3_ with easing _PARAM4_ over _PARAM5_ms as _PARAM2_"
+        ),
+        _("Size"),
+        "JsPlatform/Extensions/tween_behavior24.png",
+        "JsPlatform/Extensions/tween_behavior32.png"
+      )
+      .addParameter("object", _("Object"), "", false)
+      .addParameter("behavior", _("Behavior"), "TweenBehavior", false)
+      .addParameter("string", _("Tween Identifier"), "", false)
+      .addParameter("expression", _("To width"), "", false)
+      .addParameter("stringWithSelector", _("Easing"), easingChoices, false)
+      .setDefaultValue("linear")
+      .addParameter("expression", _("Duration"), "", false)
+      .addParameter(
+        "yesorno",
+        _("Destroy this object when tween finishes"),
+        "",
+        false
+      )
+      .setDefaultValue("no")
+      .getCodeExtraInformation()
+      .setFunctionName("addObjectWidthTween");
+
+    behavior
+      .addAction(
+        "addObjectHeightTween",
```

Comment:
Could you change the 2 actions so that they are in PascalCase (AddObjectHeightTween)? To keep consistency with other actions.